### PR TITLE
Update TiFlash related info in quickstart page

### PR DESCRIPTION
**Pull request by Joyinqin**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Update TiFlash related info in quickstart page.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/4797
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: quick-start-with-tidb.md [hunk 1]
```diff
@@ -91,7 +91,9 @@ aliases: ['/docs-cn/dev/quick-start-with-tidb/','/docs-cn/dev/how-to/get-started
 
 6. 通过 <http://127.0.0.1:2379/dashboard> 访问 [TiDB Dashboard](/dashboard/dashboard-intro.md) 页面，默认用户名为 root，密码为空。
 
-7. 测试完成后清理集群，绿色环保。通过 `ctrl-c` 停掉进程后，执行以下命令：
+7. （可选）[将数据加载到 TiFlash](/tiflash/use-tiflash.md) 进行分析。
```
Comment (TomShawn) [inline quick-start-with-tidb.md:94]:
```suggestion
7.（可选）[将数据加载到 TiFlash](/tiflash/use-tiflash.md) 进行分析。
```

File: quick-start-with-tidb.md [hunk 2]
```diff
@@ -91,7 +91,9 @@ aliases: ['/docs-cn/dev/quick-start-with-tidb/','/docs-cn/dev/how-to/get-started
...
+
+8. 测试之后，可以通过 `ctrl-c` 停掉进程并执行以下命令：
```
Comment (TomShawn) [inline quick-start-with-tidb.md:96]: 能否和中文保持一致呢
image

``` `ctrl-c` ``` --> `<kbd>ctrl</kbd> + <kbd>c</kbd>`

File: quick-start-with-tidb.md [hunk 3]
```diff
@@ -324,6 +326,10 @@ aliases: ['/docs-cn/dev/quick-start-with-tidb/','/docs-cn/dev/how-to/get-started
     - 在线部署：[使用 TiUP 部署 TiDB 集群](/production-deployment-using-tiup.md)
     - [使用 TiDB Operator 在云上部署 TiDB](https://docs.pingcap.com/zh/tidb-in-kubernetes/stable)
 
+- 如果你正在寻找 TiFlash 的分析解决方案：
```
Comment (TomShawn) [inline quick-start-with-tidb.md:329]:
```suggestion
- 如果你想使用 TiFlash 作为数据分析的解决方案，可参阅以下文档：
```

File: quick-start-with-tidb.md [hunk 4]
```diff
@@ -324,6 +326,10 @@ aliases: ['/docs-cn/dev/quick-start-with-tidb/','/docs-cn/dev/how-to/get-started
...
+    - [使用 TiFlash](/tiflash/use-tiflash.md)
+    - 查看 [TiFlash 简介](/tiflash/tiflash-overview.md)
```
Comment (TomShawn) [inline quick-start-with-tidb.md:331]:
```suggestion
    - [TiFlash 简介](/tiflash/tiflash-overview.md)
```

File: quick-start-with-tidb.md (hunk 2 above)
Comment (tisonkun) [reply quick-start-with-tidb.md:96]: +1

**Review by tisonkun — COMMENTED** [review]
Thanks for your transalation @Joyinqin ! Generally LGTM except concerns the same as @TomShawn . Please ping me if you address comments.

File: quick-start-with-tidb.md [hunk 5]
```diff
@@ -91,7 +91,13 @@ aliases: ['/docs-cn/dev/quick-start-with-tidb/','/docs-cn/dev/how-to/get-started
 
 6. 通过 <http://127.0.0.1:2379/dashboard> 访问 [TiDB Dashboard](/dashboard/dashboard-intro.md) 页面，默认用户名为 root，密码为空。
 
-7. 测试完成后清理集群，绿色环保。通过 `ctrl-c` 停掉进程后，执行以下命令：
+7.（可选）[将数据加载到 TiFlash](/tiflash/use-tiflash.md) 进行分析。
```
Comment (TomShawn) [inline quick-start-with-tidb.md:94]:
```suggestion
7. （可选）[将数据加载到 TiFlash](/tiflash/use-tiflash.md) 进行分析。
```

File: quick-start-with-tidb.md (hunk 2 above)
Comment (TomShawn) [reply quick-start-with-tidb.md]: @Joyinqin PTAL
image

**Pull request merged**